flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/plan/FlinkCalciteCatalogReaderTest.java

```diff
@@ -73,6 +73,9 @@ public void testGetCatalogSchemaTable() {
 			catalogReader.getTable(Collections.singletonList(tableMockName));
 			fail("Unexpected exception");
 		} catch (NullPointerException npe) {
+			// We have a mocked CatalogSchemaTable which does not have implementation,
```

Comment:
So what's the purpose of this test? To test an expected NPE? 
From the test method name and the original implementation, I think it want to test the expected table result.

IMO we shouldn't mock the `CatalogSchemaTable`, we can create a concrete instance of `CatalogSchemaTable`.